charts/qiskit-serverless/charts/gateway/values.yaml

```diff
@@ -27,7 +28,7 @@ application:
     maxReplicas: 4
     opensslImage: registry.access.redhat.com/ubi8/openssl:8.8-9
     kubectlImage: alpine/k8s:1.29.2@sha256:a51aa37f0a34ff827c7f2f9cb7f6fbb8f0e290fa625341be14c2fcc4b1880f60
-    proxyImage: "icr.io/quantum-public/qiskit-serverless/proxy:0.9.0"
+    proxyImage: "qiskit/qiskit-serverless-proxy:local"
```

Comment:
was this a desired change? I would maintain the old value, Aki. wdyt?

Reply:
An probably update the version.

Reply:
It's my mistake :-(